ui/tests/test_learningresources_views.py

```diff
@@ -48,6 +47,8 @@ def test_get_home_norepo(self):
         self.assertTrue("<title>MIT - LORE </title>" in body)
         self.assertFalse('<a href="/lore/create_repo/">'
                          'Create repository</a>' in body)
+        self.assertTrue('Request permission to create '
+                        'repositories</a>' in body)
```

Comment:
I guess you also need to add a test to check that the string is not there when the user has already the right to create a repo